src/JSInterop/Microsoft.JSInterop/src/IJSUnmarshalledRuntime.cs

```diff
@@ -39,7 +39,7 @@ public interface IJSUnmarshalledRuntime
         /// <param name="arg0">The first argument.</param>
         /// <param name="arg1">The second argument.</param>
         /// <returns>The result of the function invocation.</returns>
-        TResult InvokeUnmarshalled<T0, T1, TResult>(string identifier, T0 arg0, T1 arg1);
+        TResult? InvokeUnmarshalled<T0, T1, TResult>(string identifier, T0 arg0, T1 arg1);
```

Comment:
```suggestion
        TResult? InvokeUnmarshalled<T0, T1, TResult>(string identifier, T0? arg0, T1? arg1);
```

Reply:
I lean against this change as well (it shouldn't be needed, though if there is a compelling use case I would be interested to see it).